### ISSUE TITLE
[Feature Request] select parser version in browser

### ISSUE THREAD

**Issue by Amorites** [fkling/astexplorer#668]
Sometimes we need to use an old version parser for debugging purposes.

**Comment by fkling** [comment]
Yes, I understand the usefulness of this. However until now I haven't found a good way to support this, considering how many parers astexplorer supports and how the app currently works.

For this to work parsers would need to be built separately and we would need to maintain multiple configurations for different version ranges. While we should absolutely do the first (and I'm working towards this), I don't know how realistic the second one is...

My hope is that we can eventually load (some?) parsers directly from npm (via CDN) and then having a version selector would be easy to support I think.
